src/neo/Network/P2P/Payloads/TransactionVersion.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Neo.Network.P2P.Payloads
+{
+    [Flags]
+    public enum TransactionVersion : byte
+    {
+        Default = 0,
+
+        All = 0
```

Comment:
```suggestion
        All = Default
```

Reply:
It should be updated when new element is added.